### PR TITLE
Pattern stuff

### PR DESCRIPTION
**Pull request by sylvinite**
Made miseq pattern checking more explicit
Added logger case for when a samplesheet is incorrectly formatted

### CODE REVIEW

**Comment by sylvinite** [comment]
@vezzi  @Galithil Feel free to look & merge

**Comment by sylvinite** [comment]
Also fixed the problem where samples with ERROR status didn't update.

File: taca/utils/bioinfo_tab.py [hunk 1]
```diff
@@ -173,7 +176,6 @@ def get_ss_projects(run_dir):
             #This is a non platform MiSeq run. Disregard it.
             return []
         data=ss_reader.data
-
     else:
         csvf=open(FCID_samplesheet_origin, 'rU')
```
Comment (vezzi) [inline taca/utils/bioinfo_tab.py]: @sylvinite why not use samplesheet parser implemented in flowcell_parser? https://github.com/SciLifeLab/flowcell_parser/blob/master/flowcell_parser/classes.py#L213

equivalent but in this way you can reuse code

**Comment by vezzi** [comment]
fine with me, if you have time you can use flowcell_parser as suggested otherwise I can merge it

**Pull request merged**